### PR TITLE
[DEVX-1241] Add notPackage documentation

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
### Description
Updated Espresso docs to include new support of `notPackage` test option.

### Motivation and Context
Document feature in DEVX-1241.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [X] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/automated-testing/espresso-xcuitest.md
Comment (FriggaHel) [inline docs/mobile-apps/automated-testing/espresso-xcuitest.md:135]:
```suggestion
| Return additional output for troubleshooting purposes. | `--verbose` | `--verbose` |
```

**Review by FriggaHel — APPROVED** [review]

**Review by mhan83 — APPROVED** [review]
Just wondering if we need to note that `package` and `notPackage` are RDC only for now. For emulators, we are waiting on the VMD team to merge and deploy this PR: https://github.com/saucelabs/emulators-lib/pull/125

**Comment by sweeneyskirt-sl** [comment]
I can add the RDC Only badge, @mhan83. How far behind is that PR?

**Comment by sweeneyskirt-sl** [comment]
Okay @mhan83 . I added the badge and subscribed to the VMD PR. As soon as it merged, I will remove the badge.

**Pull request merged**
